go/pkg/cas/upload.go

```diff
@@ -121,6 +121,9 @@ type UploadResult struct {
 	u     *uploader
 }
 
+// ErrNoDigest indicates that the requested digest is uknown.
+var ErrNoDigest = errors.New("the requested digest is unknown")
```

Comment:
Move it up with ErrSkip?

Reply:
Could you elaborate on "skip"? 

The name "ErrSkip" is often used as a return value of a callback which tells the calling function to skip something during iteration, e.g. skip a directory and don't dive into it. Or skip a test or something.

What are we skipping here?

Reply:
Done